tests/phpunit/class-wc-stripe-helper-test.php

```
@@ -1756,4 +1756,119 @@ public function provide_test_build_line_items(): array {
 			],
 		];
 	}
+
+	/**
+	 * Resets the request-local first-gateway record so tests do not leak state.
+	 *
+	 * @return void
+	 */
+	private function reset_first_available_payment_gateway_record(): void {
+		WC_Stripe_Helper::clear_first_available_payment_gateway_record();
+	}
```

PHPMD (2.15.0)

[error] The method reset_first_available_payment_gateway_record is not named in camelCase. (undefined)

(CamelCaseMethodName)